runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkPipelineOptions.java

```diff
@@ -142,6 +142,20 @@
 
   void setNumberOfExecutionRetries(Integer retries);
 
+  @Description(
+      "Set job check interval in seconds under detached mode in method waitUntilFinish, "
+          + "by default it is 5 seconds")
+  @Default.Integer(5)
+  int getJobCheckIntervalInSecs();
+
+  void setJobCheckIntervalInSecs(int seconds);
+
+  @Description("Set the attached mode")
+  @Default.Boolean(true)
+  boolean getAttachedMode();
```

Comment:
There is a setBlockOnRun option for DirectRunner: https://github.com/apache/beam/blob/4ffeae4d2b800f2df36d2ea2eab549f2204d5691/runners/direct-java/src/main/java/org/apache/beam/runners/direct/DirectOptions.java#L36

could use the same naming

Reply:
Thanks for the comment @Abacn , I would agree to use the same naming if it is a common option in `PipelineOptions`, but IMHO it would be better to use flink-specific terminology in `FlinkPipelineOptions`

Reply:
@Abacn Any thoughts?